entity-cluster-bot/src/agent.py

```diff
@@ -1,335 +1,363 @@
 import logging
 import sys
 from datetime import datetime, timedelta
+import json
+import base64
+
 
 import forta_agent
 import networkx as nx
 import rlp
-import requests
 from forta_agent import Finding, FindingSeverity, FindingType, get_json_rpc_url
 from hexbytes import HexBytes
 from web3 import Web3
 from bot_alert_rate import calculate_alert_rate, ScanCountType
-import pickle
-import os
+
+import cProfile
+import pstats
+
+try:
+    from src.keys import ZETTABLOCK_KEY
+    from src.keys import BOT_ID
+    from src.constants import MAX_AGE_IN_DAYS, MAX_NONCE, DYNAMO_TABLE, GRAPH_KEY, ONE_WAY_WEI_TRANSFER_THRESHOLD, NEW_FUNDED_MAX_WEI_TRANSFER_THRESHOLD, NEW_FUNDED_MAX_NONCE, TX_SAVE_STEP, HTTP_RPC_TIMEOUT, PROFILING
+    from src.persistance import DynamoPersistance
+
+except ModuleNotFoundError:
+    from keys import ZETTABLOCK_KEY
+    from keys import BOT_ID
+    from constants import MAX_AGE_IN_DAYS, MAX_NONCE, DYNAMO_TABLE, GRAPH_KEY, ONE_WAY_WEI_TRANSFER_THRESHOLD, NEW_FUNDED_MAX_WEI_TRANSFER_THRESHOLD, NEW_FUNDED_MAX_NONCE, TX_SAVE_STEP, HTTP_RPC_TIMEOUT, PROFILING
+    from persistance import DynamoPersistance
+
 
 from os import environ
 
-from src.storage import get_secrets
 from dotenv import load_dotenv
 load_dotenv()
 
-SECRETS_JSON = get_secrets()
```

Comment:
this is how we can get secret keys. lets change it back. The format of the json is simple. Looks like this:

{
	"aws": {
		"key": "value"
         }
}

Reply:
sure